eng/pipelines/performance/templates/build-perf-sample-apps.yml

```diff
@@ -8,15 +8,23 @@ steps:
 # Build Android sample app
   - ${{ if eq(parameters.osGroup, 'android') }}:
     - ${{ if eq(parameters.runtimeType, 'mono') }}:
-      - script: make run TARGET_ARCH=arm64 DEPLOY_AND_RUN=false RUNTIME_FLAVOR=Mono
+      - script: |
+          make run TARGET_ARCH=arm64 DEPLOY_AND_RUN=false RUNTIME_FLAVOR=Mono > build.log
+          cat build.log
```

Comment:
Why do we need the `cat`?

Reply:
I think this is just to see the build log in the output window of the job step as well